EssentialCSharp.Web/wwwroot/css/styles.css

```diff
@@ -265,6 +265,31 @@ a:hover {
   border-color: transparent transparent var(--grey-lighten-1) transparent;
 }
 
+.sidebar-tooltip {
```

Comment:
Not positive this works totally, but can this be combined with menu-tooltip in someways to reduce duplication? 

ex:

```css
.tooltip {
  position: relative;
  display: inline-block;
}

.tooltip .tooltip-text {
  visibility: hidden;
  width: 120px;
  background-color: var(--grey-lighten-1);
  color: var(--primary-color);
  text-align: center;
  font-size: 1rem;
  padding: 5px;
  border-radius: 6px;
  position: absolute;
  z-index: 1;
  top: 120%;
  left: 50%;
  margin-left: -60px;
}

.tooltip:hover .tooltip-text {
  visibility: visible;
}

.tooltip .tooltip-text::after {
  content: " ";
  position: absolute;
  bottom: 100%;
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: transparent transparent var(--grey-lighten-1) transparent;
}

.sidebar-tooltip .tooltip-text {
  width: 150px;
  padding: 10px;
  white-space: normal;
  left: 2rem;
  bottom: -35px;
  border-radius: 10px;
}

.sidebar-tooltip:hover .tooltip-text {
  transition-delay: 1s;
  transition-property: visibility;
}
```